lib/modules/datasource/conan/types.ts

```diff
@@ -2,6 +2,14 @@ export interface ConanJSON {
   results?: Record<string, string>;
 }
 
+export interface ConanRevisionJSON {
+  revision?: string;
```

Comment:
```suggestion
  revision: string;
  time: string
```

is `revision` really optional?